### PR TITLE
update desktop message format

### PR DESCRIPTION
**Pull request by leeannachen**
fill message template with corresponding tags for datastore functions 
adjust message template for profile pictures
fix long message bug
image

### CODE REVIEW

File: proj/src/main/webapp/static/chat.html [hunk 1]
```diff
@@ -146,27 +146,41 @@ <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
                 <!-- chat display-->
                 <div class="card-body" id="message-list">
                     <!-- template for later use -->
```
Comment (aokeke) [inline proj/src/main/webapp/static/chat.html]: I think you can remove this line now, right?

File: proj/src/main/webapp/static/chat.html [hunk 2]
```diff
@@ -146,27 +146,41 @@ <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
...
+                            <div class="message"> // not added
                                 <div class="message-header">
                                 <img id="pfp">
                                 <strong><p id="username"></p></strong>
                                 <span id=timestamp></span>
                             </div>
                             <pre class="message-body"></pre>
                         </template>
-                    </div> 
+                    </div>  -->
```
Comment (aokeke) [inline proj/src/main/webapp/static/chat.html]: Can this section be deleted now?

**Review by aokeke — CHANGES_REQUESTED** [review]

File: proj/src/main/webapp/static/chat.html (hunk 1 above)
Comment (leeannachen) [reply proj/src/main/webapp/static/chat.html]: yup it's fixed now

File: proj/src/main/webapp/static/chat.html (hunk 2 above)
Comment (leeannachen) [reply proj/src/main/webapp/static/chat.html]: yup

**Review by aokeke — COMMENTED** [review]
Sorry I created conflicts that need resolving for you.

**Review by arringtonh — COMMENTED** [review]
lgtm

**Review by aokeke — APPROVED** [review]
LGTM

**Pull request merged**